include/rangers-api/Application/FSM/StateContext.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+namespace app {
+    class alignas(8) StateContext : public hh::fnd::ReferencedObject {
```

Comment:
Starting braces should be started on a line to keep consistent with the original repository.